src/pages/oncall-me/oncall-me.html

```diff
@@ -0,0 +1,90 @@
+<!--
+  Generated template for the OncallMePage page.
+
+  See http://ionicframework.com/docs/components/#navigation for more info on
+  Ionic pages and navigation.
+-->
+<ion-header>
+
+  <ion-navbar color="dark">
+    <ion-title>Me</ion-title>
+    <ion-buttons end>
+      <button ion-button icon-only (click)="openActionSheet()">
+        <ion-icon name="more"></ion-icon>
+      </button>
+    </ion-buttons>
+  </ion-navbar>
+
+</ion-header>
+
+
+<ion-content>
+  <ion-spinner *ngIf="loading" class="incident-loader">Loading...</ion-spinner>
+  <div class="user-info" *ngIf="user?.id">
+    <div class="dark">
+      <h1 class="darkText">{{ user.full_name }}</h1>
+      <h2 class="darkText"> ({{ user.name }}) </h2>
+    </div>
+
+    <ion-card>
```

Comment:
I think I might personally prefer showing an empty contact list as it indicates that you have no contacts defined and won't be interpreted as the app just not displaying contacts in general. And since there's only 3 cards I'm not too concerned with wasting screen real estate.

Reply:
We can keep showing the card, but we should include a message indicating that the user has no contact info.

Reply:
reasonable